tests/features/common/kie-controller-common.feature

```diff
@@ -15,13 +15,22 @@ Feature: KIE Controller configuration common tests
       | KIE_ADMIN_USER             | customAdm     |
       | KIE_ADMIN_PWD              | custom" Adm!0 |
       | KIE_SERVER_TOKEN           | token         |
-    Then file /opt/eap/standalone/configuration/application-users.properties should not contain customAdm
-     And file /opt/eap/standalone/configuration/application-roles.properties should not contain customAdm
-     And container log should contain -Dorg.kie.server.token=token
+    Then file /opt/eap/standalone/configuration/application-users.properties should contain customAdm=a4d41e50a4ae17a50c1ceabe21e41a80
+     And file /opt/eap/standalone/configuration/application-roles.properties should contain adminUser=kie-server,rest-all,admin,kiemgmt,Administrators,user
+
+  Scenario: Check if eap users are not being created if SSO is configured and EXTERNAL_AUTH_ONLY is not set
+    When container is started with env
+      | variable                   | value         |
+      | SSO_URL                    | http://url    |
+      | KIE_ADMIN_USER             | customAdm     |
+      | KIE_ADMIN_PWD              | custom" Adm!0 |
+    Then file /opt/eap/standalone/configuration/application-users.properties should contain customAdm=a4d41e50a4ae17a50c1ceabe21e41a80
+     And file /opt/eap/standalone/configuration/application-roles.properties should contain adminUser=kie-server,rest-all,admin,kiemgmt,Administrators,user
 
-  Scenario: Check if eap users are not being created if SSO is configured
+  Scenario: Check if eap users are not being created if SSO is configured and EXTERNAL_AUTH_ONLY is set to true
```

Comment:
For this scenario it should be also good to check also log that contains info from print_external_user_information()

Reply:
It's already there. See line 39-40